apps/photos/src/services/collectionService.ts

```diff
@@ -1024,9 +1025,11 @@ export async function getCollectionSummaries(
                 latestFile: collectionLatestFiles.get(collection.id),
                 fileCount: collectionFilesCount.get(collection.id) ?? 0,
                 updationTime: collection.updationTime,
-                type: isIncomingShare(collection, user)
-                    ? CollectionSummaryType.incomingShare
-                    : isOutgoingShare(collection)
+                type: isIncomingViewerShare(collection, user)
+                    ? CollectionSummaryType.incomingShareViewer
```

Comment:
Can we handle for newer roles? Ideally, if the RoleType is Collaborator, use incomingShareCollaborator otherwise default to Viewer